src/ansys/tools/visualization_interface/plotter.py

```diff
@@ -56,6 +56,31 @@ def plot(self, object: Any, **plotting_options):
         """
         self._backend.plot(object=object, **plotting_options)
 
-    def show(self):
-        """Show the plotted objects."""
-        self._backend.show()
+    def show(
+        self,
+        object: Any = None,
+        screenshot: str = None,
+        view_2d: bool = None,
+        filter: bool = None,
+        **plotting_options
+        ) -> None:
+        """Show the plotted objects.
```

Comment:
Seems a bit PyVista specific, right?

Reply:
I would say maybe for the view_2d parameter, but having screenshots should be common for all backends, name filtering is a feature of this tool, and generic plotting options should be available in other backends too

Reply:
Let's abstract it out then and allow for view_2d to be passed as kwargs/plotting options. I like the other though.

Reply:
I.e. I think that in that case you just need to remove the "view_2d" arg. It will be extracted from plotting_options if it is given ( I believe )

Reply:
Will do, makes total sense

Reply:
Done, works fine with Geometry